_aggregations/metric/minimum.md

```diff
@@ -9,30 +9,46 @@ redirect_from:
 
 # Minimum aggregations
 
-The `min` metric is a single-value metric aggregations that returns the minimum value of a field.
+The `min` metric is a single-value metric that returns the minimum value of a field.
 
-The following example calculates the minimum of the `taxful_total_price` field:
+The `min` aggregation compares numeric fields using a `double` (double-precision) representation. Results should be considered approximate for fields containing only `long` or `unsigned_long` integer values greater than 2^53, since the number of significant bits in a `double` mantissa is 53.
```

Comment:
Even less likely than seeing this effect in `max`. (See above). Please comment.

Reply:
This seems like a reasonable warning nonetheless. It could matter if you have very negative numbers.

I think if we include the warning on the max side it makes sense to mirror it on the min side.

Reply:
@msfroh Good point about the negative numbers. I was fixated on large positive numbers, probably because I included `unsigned_long`. But large negative `long` values would pose a problem as well.